internal/controller/datadogagent/feature/enabledefault/feature_test.go

```diff
@@ -69,22 +69,6 @@ func Test_getInstallInfoValue(t *testing.T) {
 	}
 }
 
-func Test_defaultFeature_ManageClusterAgent(t *testing.T) {
-	tests := test.FeatureTestSuite{
-		{
-			Name: "Manage Cluster Agent service account name env variable",
-			DDA: testutils.NewDatadogAgentBuilder().
-				WithName("datadog").
-				WithEventCollectionKubernetesEvents(true).
-				Build(),
-			WantConfigure: true,
-			ClusterAgent:  test.NewDefaultComponentTest().WithWantFunc(defaultFeatureManageClusterAgentWantFunc),
```

Comment:
`defaultFeatureManageClusterAgentWantFunc` is not used anymore and can be deleted